finality-aleph/src/party/mod.rs

```diff
@@ -396,6 +405,11 @@ where
         if let Err(e) = self.session_manager.stop_session(session_id) {
             warn!(target: "aleph-party", "Session Manager failed to stop in session {:?}: {:?}", session_id, e)
         }
+
+        {
```

Comment:
`unit_saving_path` here has to be cloned and the clone moved into the closure, so that the closure doesn't borrow anything (`spawn_blocking()` requires a `'static` argument). Scoping closure setup feels cleaner/natural to me; do you think something else might be better? 😄

Reply:
yeah, I get this requirement (you can see legendary `do_async!` macro in `traffic-maker` :joy:), but it seems to me that apart from cloning you don't need to put it in a new block

Reply:
oh, okay, I get it, it's just a hygiene, isn't it? if so, then I'm okay with that and you can resolve this discussion - until now I was thinking that it's necessary